tamagotchi-01/tests/tamagotchi_test.rs

```diff
@@ -0,0 +1,15 @@
+#![no_std]
+use gtest::{Log, Program,System};
+use gstd::String;
+
+#[test]
+fn tamagotchi_test() {
+    let sys = System::new();
+    sys.init_logger();
+    let program = Program::current(&sys);
+
+    let response = program.send(2, String::from("INIT MESSAGE"));
+    let expected_log = Log::builder().dest(3).payload(String::from("123"));
```

Comment:
We don't expect a reply message from the `init` function. Also, you are to call one of the `assert!`/`assert_eq!`/`assert_ne!`/etc. macros in test functions.

Reply:
are you the teacher of the academy?